packages/picasso/src/PageHead/PageHead.tsx

```diff
@@ -28,36 +28,45 @@ const useMainStyles = makeStyles(styles, {
 const Title = ({
   titleCase,
   children,
-}: TextLabelProps & { children: ReactNode }) => {
+  className,
+}: TextLabelProps & { children: ReactNode } & BaseProps) => {
   return (
-    <Typography variant='heading' size='large' titleCase={titleCase}>
+    <Typography
+      variant='heading'
+      size='large'
+      titleCase={titleCase}
+      className={className}
```

Comment:
```suggestion
      className={className}
      {...rest}
```
Like in other places when we use `BaseProps`:
https://github.com/toptal/picasso/blob/e937354d9a93f4a02c91e4daa5ca4543535c58d6/packages/picasso/src/Menu/Menu.tsx#L17-L54